Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1

```diff
@@ -1,108 +1,161 @@
+### Convert Params to match vars
+$app_baseUrl = $AppBaseURL
+$app_apikey = $AppAPIKey
+## Start Variables
+### Application Details
+$app_api_version = 'v1'
+$app_api_path = '/api/'
+$app_api_endpoint_version = '/system/status'
+$app_api_endpoint_indexer = '/indexer/schema'
+$headers = @{'X-Api-Key' = $app_apikey }
+### Github App Info
+$gh_app_org = 'Prowlarr'
+$gh_app_repo = 'Prowlarr'
+### Wiki Details
+$wiki_link = 'https://wiki.servarr.com'
+$wiki_app_path = '/prowlarr'
+$wiki_page = 'supported-indexers'
+$wiki_bookmark = '#'
+### Page Formating
+$wiki_1newline = "`r`n"
+$wiki_2newline = "`r`n`r`n"
+$wiki_encoding = 'utf8'
+### Github Details
+$gh_web = 'https://github.com'
+$gh_web_commit = 'commit/'
+## End Variables
+
+Write-Information 'Variables and Inputs Imported'
+## Build Parameters
+### App
+$api_url = ($app_baseUrl.ToString().TrimEnd('/')) + $app_api_path + $app_api_version
+$version_url = $api_url + $app_api_endpoint_version
+$indexer_url = $api_url + $app_api_endpoint_indexer
+### Github
+$gh_repo_org = $gh_app_org + '/' + $gh_app_repo + '/'
+### Wiki
+$wiki_infolink = ($wiki_link.ToString().TrimEnd('/')) + $wiki_app_path + '/' + $wiki_page + $wiki_bookmark
+$wiki_commiturl = ($gh_web.ToString().TrimEnd('/')) + '/' + $gh_repo_org + $gh_web_commit
+Write-Information 'Parameters Built'
+
+## Invoke Requests & Convert to Objects
+Write-Information 'Getting Version Data and Converting Response to Object'
+$version_obj = (Invoke-WebRequest -Uri $version_url -Headers $headers -ContentType 'application/json' -Method Get).Content | ConvertFrom-Json
+Write-Information 'Got App Version'
+Write-Information 'Getting Indexer Data and Converting Response to Object'
+$indexer_obj = (Invoke-WebRequest -Uri $indexer_url -Headers $headers -ContentType 'application/json' -Method Get).Content | ConvertFrom-Json
+Write-Information 'Got Indexer Data'
+$indexer_name_exp = { IF ($_.IndexerUrls) { '[' + $_.name + '](' + ($_.IndexerUrls[0]) + ')' + '{#' + $_.infoLink.Replace($wiki_infolink.ToString(), '') + '}' } Else { $_.name + '{#' + $_.infoLink.Replace($wiki_infolink.ToString(), '') + '}' } }
+
+## Determine Commit
+Write-Information "Commit is $commit"
+
+## Determine Version (Build)
+Write-Information 'Determining Build'
+$build = $version_obj | Select-Object -ExpandProperty 'version' | Out-String | ForEach-Object { $_ -replace "`n|`r", '' }
+Write-Information "Build is $build"
+
+Write-Information 'Ingesting Indexer Data'
+## Get Indexer Data
+$indexer_tbl_obj = $indexer_obj | Sort-Object -Property 'name'
+
+## Build Table Fields
+Write-Information 'Building Indexer Tables'
+### Public Usenet
+Write-Information 'Building: Usenet - Public'
+$tbl_PubUse = $indexer_tbl_obj | Where-Object { ($_.privacy -eq 'public') -and ($_.protocol -eq 'usenet') } | `
+    Select-Object @{Name = 'Indexer'; Expression = $indexer_name_exp }, `
+@{Name = 'Language'; Expression = { $_.language } }, `
+@{Name = 'Description'; Expression = { $_.description } }
```

Comment:
Ohh also, I don't like backticks so get rid and make them long lines - unless @Qstick, @ta264 or @austinwbest have issues with very long lines (like these ones and the other 3  similar will become)

Reply:
They are generally frowned upon in the PowerShell community as well